ruby_on_rails/app_initialisation.md

```diff
@@ -94,6 +94,17 @@ create the initializer for figaro in `config/initializers/figaro.rb`:
 
 * Run `bin/setup` again.
 
+### Secrets
+
+We store the secrets necessary to configure the project locally in a 1password vault.
+Create a new vault for the project called `[project-name] project secrets` of type note.
+Right click on the vault and select `Copy Private link`.
```

Comment:
instead of a new vault called `[project-name] project secrets`, I think it would be nice to have a vault per project. @schmijos and I made this an operations goal for this year. See https://docs.google.com/document/d/1kqig4ITDiSciXu6mGtIg8b0bUsqQXjNl8De6gu5Y3Hg/edit?tab=t.0#heading=h.tr0aaqqpaniu

@coorasse what do you think about this?

Reply:
I don't see the need to have a separate vault per customer. 

But when we will have, we can of course change this part here as well 👍